interpreter/statement.go

```diff
@@ -430,7 +430,7 @@ func (i *Interpreter) ProcessIfStatement(stmt *ast.IfStatement, ds DebugState) (
 			return state, nil
 		}
 	case *value.String:
-		if t.Value != "" {
+		if !t.IsNotSet {
```

Comment:
Shouldn't the "else if" section have this change too? https://github.com/ysugimoto/falco/blob/105b9118206a212abc75144af629839130f8839d/interpreter/statement.go#L469-L471

Reply:
Good point. I forget to fix here, thanks

Reply:
Fixed it, and also fixed in if expression.